litellm/llms/bedrock/chat/converse_transformation.py

```diff
@@ -87,10 +87,11 @@ def get_supported_openai_params(self, model: str) -> List[str]:
             or model.startswith("mistral")
             or model.startswith("cohere")
             or model.startswith("meta.llama3-1")
+            or model.startswith("us.anthropic")
```

Comment:
it would probably be safer to strip the region from the model name before entering this branch, since this code will also need to account for `eu.` etc.

Reply:
Amazing, I just did a quick fix to get us running 👍